### ISSUE TITLE
iD editor version upgrade 

### ISSUE THREAD

**Issue by ramyaragupathy** [hotosm/tasking-manager#5780]
We are still using a somewhat outdated version of iD ([v2.21.1](https://github.com/hotosm/tasking-manager/blob/develop/frontend/package.json#L11) – the most recent version of iD is v2.25.2) in the tasking manager. 

cc @eternaltyro @emi420 @Aadesh-Baral @HelNershingThapa

**Comment by ramyaragupathy** [comment]
This should also cover the maxar key update - which was applied as a hotfix through a temporary solution.

**Comment by cyrossignol** [comment]
Hello, I updated iD to version 2.26.2 for our use at [tasks.opensidewalks.com](https://tasks.opensidewalks.com/) here: OpenSidewalks@786c5108b03713463ed77194be13638ef03fd4a8

I reworked the integration by avoiding the direct bundling of iD code with the tasking manager JS. This gets around an issue with the output of iD's newer esbuild bundling system and significantly improves the tasking manager build time. I'm inclined to apply the same treatment for RapiD to remove the need for the special upstream releases.

Let me know if this is worth a PR or if HOT is pursuing a different upgrade strategy.

**Comment by ramyaragupathy** [comment]
@cyrossignol - thanks for checking on this. Your idea sounds good, could you please go ahead making a PR for our team to go through this in detail?

cc @HelNershingThapa @tsmock

**Comment by tsmock** [comment]
I'll have to check and see how well it works with Rapid. Of specific note, I don't know if waiting for Rapid to finish "loading" (via `script.onload`) will work -- Rapid does a lot of setup async now (current `main`, should be released sometime soon).

I haven't touched the iD code, but iD also doesn't have as much backend changes going on.

**Comment by tsmock** [comment]
I've been playing around with the `script` loading stuff since Rapid v2.1 was released.

Some comments on how it improved the build experience and could (potentially) make it much easier to update out-of-band:
* We can use a CDN (e.g. `https://cdn.jsdelivr.net/npm/@rapideditor/rapid@~2.1.0/dist/rapid.js`)
 * Pro: Reuse of rapid download between TM instances
 * Con: Dependent upon free CDN
* It _significantly_ reduces the build time for me -- as in 18 minutes to 3 minutes for me.

If anyone wants to play with that (for Rapid), I've got a branch here: https://github.com/facebook/OSM-HOT-Tasking-Manager/tree/chore/update-rapid

Notes:
* I'm debugging an issue with task switching; if you start a task, then switch to a different task, Rapid doesn't reload. The work around is a hard refresh. I have another (code) workaround, but it makes the Rapid UI jittery (I think two input events are getting fired for the same input).
